src/main/ts/ScriptLoader.ts

```diff
@@ -24,20 +25,23 @@ const createState = (): IStateObj => {
 };
 
 interface ScriptLoader {
-  load: (doc: Document, url: string, callback: callbackFn) => void;
+  load: (doc: Document, url: string, callback: callbackFn, loadTimeout?: number, scriptAttributes?: IProps['scriptAttributes']) => void;
   reinitialize: () => void;
 }
 
 const CreateScriptLoader = (): ScriptLoader => {
   let state: IStateObj = createState();
 
-  const injectScriptTag = (scriptId: string, doc: Document, url: string, callback: callbackFn) => {
-    const scriptTag = doc.createElement('script');
+  const injectScriptTag = (scriptId: string, doc: Document, url: string, callback: callbackFn, scriptAttributes?: IProps['scriptAttributes']) => {
+    let scriptTag = doc.createElement('script');
     scriptTag.referrerPolicy = 'origin';
     scriptTag.type = 'application/javascript';
     scriptTag.id = scriptId;
     scriptTag.src = url;
 
+    if (scriptAttributes)
```

Comment:
We generally avoid if statements that don't have brackets (ie we want `if (...) {`) though we sometimes allow for single lines.